src/app/users/users.component.ts

```diff
@@ -49,29 +49,19 @@ export class UsersComponent implements OnInit {
     return this.couchService.get('_users/_all_docs?include_docs=true');
   }
 
-  getAdmins() {
-    // This nonode@nohost is working for couchdb as setup by Vagrant, but may need to be changed for other implementations
-    return this.couchService.get('_node/nonode@nohost/_config/admins');
-  }
-
   initializeData() {
     forkJoin([

```

Comment:
You can remove this `forkJoin` since there is only one observable (`this.getUsers()`) now